chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -39,21 +39,14 @@ def runningAsAdministrator(self):
         capabilities, but here we are testing the capabilities itself so is
         kind of chicken and egg problem.
         """
-        # Windows 2008 and DC client tests are done in administration mode,
-        # 2003 and XP under normal mode.
-        if 'win-2003' in self.hostname or 'win-xp' in self.hostname:
-            return False
+        admin = False
+        try:
+            admin = os.getuid() == 0
+        except AttributeError:
+            import ctypes
```

Comment:
I think that this ctypes import can be top level, as all our python versions should have ctypes available